public/editor/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -1,314 +1,78 @@
 define(function(require) {
```

Comment:
Can we add a comment block at the top to explain the purpose of this module? Something indicating that this is a high-level API.